example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,39 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+The main branch contains only the code needed for the tutorial. Results of each step can be found in tags, and resulting state in correctponding branches.
```

Comment:
please apply the standard formatting (80 symbols, etc) ... I need to check if pre-commit is setup for this repo

Reply:
Done.